db/seeds.rb

```diff
@@ -1 +1,20 @@
+require 'faker'
 
+1.times do
+  User.create!(
+    email: Faker::Internet.email,
+    encrypted_password: Faker::Internet.password
```

Comment:
I changed `encrypted_password` to simply `password` and that fixed my immediate problem.  Then it was just a matter of cleaning a couple of other things up.